llvm/lib/Target/AIE/AsmParser/AIE1AsmParser.cpp

```diff
@@ -0,0 +1,512 @@
+//===-- AIEAsmParser.cpp - Assembly Parser for AIE -------- ------*- C++ -*--//
```

Comment:
AIE`1`AsmParser.cpp...

Is this file a copy of the AIE2AsmParser.cpp ? If so. can we keep code common?

Reply:
unfortunately there are a number of minor differences which are somewhat difficult to factor out.